nuklear.h

```diff
@@ -5927,7 +5935,7 @@ struct nk_text {
     struct nk_color text;
 };
 NK_LIB void nk_widget_text(struct nk_command_buffer *o, struct nk_rect b, const char *string, int len, const struct nk_text *t, nk_flags a, const struct nk_user_font *f);
-NK_LIB void nk_widget_text_wrap(struct nk_command_buffer *o, struct nk_rect b, const char *string, int len, const struct nk_text *t, const struct nk_user_font *f);
+NK_LIB void nk_widget_text_wrap(struct nk_command_buffer *o, struct nk_rect b, const char *string, int len, const struct nk_text *t, const struct nk_user_font *f, int do_wrap);
```

Comment:
Having the name `nk_widget_text_wrap()`, one would assume it should wrap, regardless of the parameter. In order to avoid the breaking change, could we have an `if` statement prior to the call and call either `nk_widget_text()` or `nk_widget_text_wrap()` based on whether the wrap is required?